IORawData/CaloPatterns/src/HtrXmlPattern.cc

```diff
@@ -6,10 +6,42 @@
 #include "DataFormats/HcalDigi/interface/HcalDigiCollections.h"
 #include "CondFormats/HcalObjects/interface/HcalElectronicsMap.h"
 
-#include "HtrXmlPattern.h"
 #include "HtrXmlPatternTool.h"
 #include "HtrXmlPatternToolParameters.h"
 
+// system include files
+#include <memory>
+
+// default include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "CondFormats/HcalObjects/interface/HcalElectronicsMap.h"
+
+class HtrXmlPattern : public edm::one::EDAnalyzer<> {
+public:
+  explicit HtrXmlPattern(const edm::ParameterSet&);
+  ~HtrXmlPattern() override;
+
+private:
+  void analyze(const edm::Event&, const edm::EventSetup&) override;
+  void endJob() override;
+  virtual void do_hand_fill(const HcalElectronicsMap*);
+  HtrXmlPatternTool* m_tool;
+  HtrXmlPatternToolParameters* m_toolparameters;
+  int m_sets_to_show;
+  int m_hand_pattern_number;
+  bool m_fill_by_hand;
+  bool m_filled;
+  bool m_write_root_file;
+
+  const edm::ESGetToken<HcalDbService, HcalDbRecord> theHcalElectronicsMapToken_;
```

Comment:
To maintain the same style for all class members, this could be `m_hcalElectronicsMapToken` (very minor point indeed...)

Reply:
Yeah I was thinking about this... but the trailing underscore is the CMS recommendation and not the `m_*` so kind of like everything else is what would need to be changed, not this line. :)

Reply:
Fine, since I'm touching the code, I can do the not-CMS-recomended version to be internally consistent with the code style here